### ISSUE TITLE
Questions about the 'multiscreen' option in SceneSummaryNode

### ISSUE THREAD

**Issue by samreid** [phetsims/scenery-phet#379]
While working on https://github.com/phetsims/color-vision/issues/121 I noticed:

```js
  function SceneSummaryNode( sceneSummary, options ) {
    // ...
    options = _.extend( {
      // ...
      multiscreen: true // to use the default multiscreen intro or single screen intro
    }, options );
```

I was curious about this multiscreen flag.  Please correct me if I'm wrong, but as I understand it, there is one SceneSummaryNode per Screen (I think @pixelzoom pointed out the inconsistency in this nomenclature).  So each screen would begin with this text: `'This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. '`

It seems this would lead to the following pdom:
```html
<div id="accessibility">
<section id="screen1">
'This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. '
</section>
</div>
```

But it seems the sim should be a container for the screens, and the single/multiscreen text would be associated with the SimView which we have discussed creating in Joist.  Then the pDOM would be something like this:

```html
<div id="accessibility">
<section id="simulation">
'This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. '
  <section id="screen1"> ... </section>
</section>
</div>
```

That is, the "this is a sim" message should appear once at the "sim" level instead of getting copied into each ScreenView.  This will also help for the case where one screen of a multiscreen sim is used as a single screen sim (like Concentration).

**Comment by pixelzoom** [comment]
I agree @samreid.  This summary seems like a mix of "sim" and "screen" information.

**Comment by mbarlow12** [comment]
One consideration is that, in the second scenario, users' assistive technology is presented with 2 distinct areas in the html where sim/screen summary information is provided. Since we have the ability to hide the different screens' accessible content (e.g. the entire `<section id="section-1"></section>` element), the first example provides users with a single place where users can quickly access that same information, albeit with some repeated content.

**Comment by mbarlow12** [comment]
I think the idea is to have the first accessible content on an given 'screen' provide the user with the summary info. While it may be an artifact of instrumenting single-screen sims, I think it's still useful to the user to provide a bit of sim info and screen info when they switch screens. It seems like from a user's perspective, having both information always available would be helpful; however, this could be researched with user interviews.

Something else to consider is that container elements with no `aria-label` or `aria-describedby`/`aria-labeledby` attributes don't affect how screen readers interpret the DOM. The following should produce the same output from a screen reader:

```html
<div>
    <h3>A Heading</h3>
    <section id="summary">
        <p>This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. <p>
    </section>
</div>


<div>
    <h3>A Heading</h3>
    <p>This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. <p>
</div>


<h3>A Heading</h3>
<p>This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel. <p>
```

So we may be able to achieve the same goal by targeting a structure like:
```html
<div id="accessibility">
    <h2>Scene Summary</h2>
    <p>This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel.<p>
        <!-- start screen1 specific summary content -->
    <p>Screen 1 has some cool stuff to do. The features are: </p>
    <ul>
        <li>a value</li>
        <li>another value</li>
        <li>a third value</li>
    </ul>
    <section id="screen1">
        <!-- html generated from sim nodes -->
    </section>

    <!-- start screen2 specific summary content -->
    <p>Screen 2 is simpler than screen 1. There's only a 'p' tag here </p>
    <section id="screen2">
        <!-- html generated from sim nodes -->
    </section>
    ...
    ...
    ...
</div>
```
It may also be possible to create an opening `section` tag that is closed in each of the screens with different content included:
```html
<div id="accessibility">
    <section id="sim-screen-summary">
        <h2>Scene Summary</h2>
        <p>This is an interactive sim. It changes as you play with it. Each screen has a Play Area and Control Panel.<p>

        <!-- start screen1 specific summary content -->
        <p>Screen 1 has some cool stuff to do. The features are: </p>
        <ul>
            <li>a value</li>
            <li>another value</li>
            <li>a third value</li>
        </ul>
    </section>

    <section id="screen1"> <!-- active/selected screen -->
        <!-- html generated from sim nodes -->
    </section>

        <!-- ***HIDE ALL CONTENT FROM HERE TO screen2's CLOSING TAG*** -->

        <!-- start screen2 specific summary content -->
        <p>Screen 2 is simpler than screen 1. There's only a 'p' tag here </p>
    </section>
    <section id="screen2">
        <!-- html generated from sim nodes -->
    </section>
    ...
    ...
    ...
```

**Comment by zepumph** [comment]
I think since this issue came to life, we are in the process of designing away from a "scene summary" section, but the content will still remain. 

Assigning to @terracoda to help us design what this will look like. I think this is potentially a duplicate of https://github.com/phetsims/a11y-research/issues/104.

**Comment by terracoda** [comment]
Great discussion folks. @zepumph is correct, without the first "scene summary" section, the overall structure is going to change slightly. The most significant change being that we will not use a section element - a section element requires a heading to be well-formed HTML.  I should be clarifying the structure this week.

I would like to point out that I have given considerable consideration to how we would potentially handle multi-screen sims when I was working on Build and Atom. That work was shelved, so not many people have seen the initial ideas that were used in that simulation.

As @mbarlow12 mentioned above, it is indeed essential to set up the scene separately for each "screen". And additionally, I think that the little opening blurb should mention that the user is on one of X number of screens.

The screens have an order to follow, albeit an optional order, in Build an Atom, I numbered and named the screens as a way of scaffolding the order but not enforcing it.

**Comment by pixelzoom** [comment]
@terracoda said:
> The screens have an order to follow, albeit an optional order, in Build an Atom, I numbered and named the screens as a way of scaffolding the order but not enforcing it.

You may already be considering this, but I'll mention here.  Screens frequently appear in more than one sim.   So if there's anything I'm the description that is order specific, that may prove problematic when the screen appear in a different sim, in (possibly) a different order.

**Comment by terracoda** [comment]
@pixelzoom, thanks for mentioning this. I was not aware that the same screen could be used in multiple sims. That is good to know! 

It would make sense to have the name and number of the screen be dynamic parameters.

**Comment by mbarlow12** [comment]
@zepumph @terracoda while we may no longer be calling it a 'scene summary' or using a `<section>` tag, it is my understanding that each screen would still have an html element (or a few) that offered summarized information about the whole sim/screen. If that's no longer the case, please correct me.

**Comment by terracoda** [comment]
@mbarlow12, yes that is correct. There is still an opening introduction for each screen, but without the Scene Summary section.

**Comment by terracoda** [comment]
@samreid and @pixelzoom have your questions and concerns been answered/addressed? Does the HomeScreen.md file clarify why a flag is needed?

I think single screen sims should begin like this:
```html
<h1>{{Sim Name}}</h1>
<p>{{Sim Name}} is an interactive sim. It changes as you play with it. It has a Play Area and a Control Area. [...] </p>
```

And a single screen in a multi-screen sim should start like this:

```html
<h1>{{Screen Name, Sim Name}}</h1>
<p>The interactive {{Screen Name}} Screen changes as you play with it. It has a Play Area and a Control Area. [...] 
```

The curly braces represent content that is specific to the sim.

**Comment by terracoda** [comment]
@samreid and @pixelzoom, please re-assign if you have more questions.

**Comment by terracoda** [comment]
Or perhaps this issue can be closed?

**Comment by samreid** [comment]
The comment in https://github.com/phetsims/scenery-phet/issues/379#issuecomment-404593852 seems to have addressed the problem discussed in https://github.com/phetsims/scenery-phet/issues/379#issue-322158916

**Comment by pixelzoom** [comment]
@terracoda asked:
> @samreid and @pixelzoom have your questions and concerns been answered/addressed? Does the HomeScreen.md file clarify why a flag is needed?

No, not obviously.  I see no mention of "multiscreen" or "flag" in HomeScreen.md.

Other questions that have not been addressed:
• Is the summary still a mix of sim and screen info? If so why? (asked in https://github.com/phetsims/scenery-phet/issues/379#issuecomment-388253431)
• How are we addressing the fact that screens may appear in >1 sim?  (noted in https://github.com/phetsims/scenery-phet/issues/379#issuecomment-392534765)
• Has the terminology been changed to avoid the confusing/contradictory usage of "scene"? (noted in https://github.com/phetsims/scenery-phet/issues/379#issue-322158916 and https://github.com/phetsims/scenery-phet/issues/379#issuecomment-393990662)

**Comment by terracoda** [comment]
@pixelzoom, I'm not sure I can address all the details, but together with the a11y devs, I think all these questions will be answered.
- I assume the flag will be something built into the A11y API ( I defer to @jessegreenberg on that). The idea being, if you are building a single screen sim, you use one description pattern, and if you are designing a multi-screen sim you use another description pattern.
- The summary is not a mix of just any sim content and screen content. It is a summarized description of the content that is currently loaded (i.e., the screen). There will be parameters for the Sim Name, the Screen Name and the Number of Screens.
- Screens may appear in different sims, and the parameters for teh Sim Name and the Screen Name should handle these scenarios.
- Going forward, there is no more use of the heading, "Scene Summary", and I have not used the term "scene" in the latest design template.

I hope these answers address your concerns.

**Comment by terracoda** [comment]
Awesome! Thanks @pixelzoom for closing :-)
